File: specification/specs/x_pack/sql/query_sql/QuerySqlRequest.ts
@rest_spec_name("sql.query")
class QuerySqlRequest extends RequestBase {
  query_parameters?: {
    format?: string;
  }
  body?: {
    columnar?: boolean;
    cursor?: string;
    fetch_size?: integer;
    filter?: QueryContainer;
    query?: string;
    time_zone?: string;
  }
}

File: output/csharp/XPack/Sql/QuerySql/QuerySqlRequest.cs
using Nest.Internal;
using Nest.QueryDsl;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class QuerySqlRequest : RequestBase {
		
		[DataMember(Name="format")]
		public string Format { get; set; }


		[DataMember(Name="columnar")]
		public bool Columnar { get; set; }


		[DataMember(Name="cursor")]
		public string Cursor { get; set; }


		[DataMember(Name="fetch_size")]
		public int FetchSize { get; set; }


		[DataMember(Name="filter")]
		public QueryContainer Filter { get; set; }


		[DataMember(Name="query")]
		public string Query { get; set; }


		[DataMember(Name="time_zone")]
		public string TimeZone { get; set; }

	}
}
